### PR TITLE
Candidate for v0.0.1

### PR DESCRIPTION
**Pull request by olajideoye**
Here's the candidate for version 0.0.1
- Plugin can be instantiated through HTML and jQuery, and options can be passed in through both ways
- Linting (JSHint) added to aid development
- Build tool (Gulp.js) added

### CODE REVIEW

**Comment by olajideoye** [comment]
Next steps after merge would be updating the version number to 0.0.1 and tagging.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,93 @@
...
+```html
+<a href="ajax.html" data-mfp data-mfp-options='{"type": "ajax"}'>Here's a Magnific Popup link</a>
+```
+
+Please note that the content of the `data-mfp-options` attribute must be valid JSON to work.
+Something like ` data-mfp-options='{"type": "ajax", "modal": true}'` is valid and ` data-mfp-options="{'type': 'ajax'}"` is invalid.
+
+
+#### Stop propagation on child elements
+To prevent the popup from opening on children elements of a Magnific Popup link, use the `data-mfp-no-propagate` attribute.
```
Comment (yemexx1) [inline README.md]: Is there a default value for this. If there is, it will be good to state it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+    eventNamespace: '.magnific-popup-wrapper',
+
+    /* data to be passed into Magnific Popup */
+    options: {}
+}
+```
+
+
+
+## Zepto.js support?
```
Comment (yemexx1) [inline README.md:88]: Is this true?

Comment (olajideoye) [inline README.md:88]: Zepto.js is supported for the Magnific Popup itself. I'm just stating that it isn't for the wrapper

Comment (yemexx1) [inline README.md:88]: Okay. Nice 👍

**Pull request merged**